setup/README.md

```diff
@@ -11,30 +11,64 @@ The main steps are:
 
 3. Download and install [Google Cloud SDK](https://cloud.google.com/sdk/docs/).
 
-4. Configure the SDK by running `gcloud init` and following the instructions.
+4. Configure the SDK by running:
+
+From your terminal:
+
+   ```shell
+   gcloud init
+   ```
+   and follow the instructions.
 
 5. Enable the API for the following services:
 
-  * [Compute Engine](https://pantheon.corp.google.com/compute). Run `gcloud services enable compute.googleapis.com`.
-  * [Storage](https://pantheon.corp.google.com/storage). Run `gcloud services enable storage-component.googleapis.com`.
-  * [AI Platform](https://pantheon.corp.google.com/mlengine). Run `gcloud services enable ml.googleapis.com`.
+  * [Compute Engine](https://pantheon.corp.google.com/compute).
+  * [Storage](https://pantheon.corp.google.com/storage).
+  * [AI Platform](https://pantheon.corp.google.com/mlengine).
+
+From your terminal:
+
+```bash
+gcloud services enable compute.googleapis.com
+gcloud services enable storage-component.googleapis.com
+gcloud services enable ml.googleapis.com
+```
 
-6. Create and download a service account key with the right permissions, by following the instructions [here](https://cloud.google.com/iam/docs/creating-managing-service-account-keys#iam-service-account-keys-create-gcloud).
+6. Create and download a service account key with the right permissions, follow the instructions [here](https://cloud.google.com/iam/docs/creating-managing-service-account-keys#iam-service-account-keys-create-gcloud).
 
-7. Open `setup.sh` in this directory. Set the following variables:
+7. Edit `setup.sh` in this directory. Set the following variables:
 
   - `PROJECT_ID`
   - `BUCKET_NAME` 
   - `GOOGLE_APPLICATION_CREDENTIALS` 
  
-  Then run 
+From your terminal:
  
- ```bash
+```bash
  source ./setup.sh
- ```
+```
+
+Or you can copy and paste the following with the corresponding edits.
```

Comment:
Nor sure how to format this in a README to make it clear that there are two options. 

Maybe like 
```
From your terminal choose from the following two options:
1. bash..
2. Export ...
```